lib/rules/require-prop-comment.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @author CZB
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    type: 'problem',
```

Comment:
I think this rule is `"suggestion"`.

```suggestion
    type: 'suggestion',
```

Reply:
o'kok